webcomponents/tool/src/main/frontend/js/rubrics/sakai-item-delete.js

```diff
@@ -61,15 +61,15 @@ export class SakaiItemDelete extends RubricsElement {
         <span class="fa fa-times"/>
       </button>
 
-      <div id="delete_${this.type}_${this.item.id}" class="popover rubric-delete-popover left">
-        <div class="arrow"></div>
+      <div id="delete_${this.type}_${this.item.id}" class="popover rubric-delete-popover left" style="top: 389px; left: -10px; display: none; position: absolute; padding: 10px;">
```

Comment:
You should move these inline styles into rubric-delete-popover.

Reply:
Thanks :)